doc/whats_new.rst

```diff
@@ -27,6 +27,8 @@ API
 
     - Added options for setting data and date formats manually in :func:`mne.io.read_raw_cnt` by `Jaakko Leppakangas`_
 
+    - Now it is possible to plot only a subselection of channels in :func:`mne.viz.plot_raw` by using an array for order parameter by `Jaakko Leppakangas`_
```

Comment:
I don't think we need a note here, the above one should suffice. (IMO this section is really more for API changes that would cause people to change existing code for future compatibility than for additions of functionality.)